public/pages/workflows/new_workflow/new_workflow.tsx

```diff
@@ -65,28 +122,36 @@ export function NewWorkflow(props: NewWorkflowProps) {
     dispatch(searchConnectors({ apiBody: FETCH_ALL_QUERY, dataSourceId }));
   }, []);
 
-  // initial hook to populate all workflows
-  // enrich them with dynamically-generated UI flows based on use case
```

Comment:
nit: still prefer to leave these as possible, helpful to refer to them

Reply:
addressed on line 133